pkg-manager/core/src/install/index.ts

```diff
@@ -1004,7 +1004,8 @@ const _installInContext: InstallFunction = async (projects, ctx, opts) => {
   } = await resolveDependencies(
     projects,
     {
-      allowBuild: createAllowBuildFunction(opts),
+      // In the next major allow build should be just () => true here always
+      allowBuild: opts.onlyBuiltDependenciesFile ? () => true : createAllowBuildFunction({ onlyBuiltDependencies: opts.onlyBuiltDependencies, neverBuiltDependencies: opts.neverBuiltDependencies }),
```

Comment:
I might be missing something. If `opts.onlyBuiltDependenciesFile` is non-null, we'll want to call `createAllowBuildFunction`, is that right? It currently evaluates to `() => true` instead.

Or is this intentional and the usage of `opts.onlyBuiltDependenciesFile` opts into the default behavior in the next major version, which would be to always return `() => true`?

Reply:
> Or is this intentional and the usage of opts.onlyBuiltDependenciesFile opts into the default behavior in the next major version, which would be to always return () => true?

yes, correct